docs/diagrams/ShortcutSettingsClassDiagram.puml

```diff
@@ -0,0 +1,20 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Class AddressBookParser LOGIC_COLOR
+Class "<<interface>>\nModel" as Model
+Class ModelManager
+Class UserPrefs
+Class ShortcutSettings
+Class GuiSettings
+
+AddressBookParser -[LOGIC_COLOR]right-> "1" ModelManager
+ModelManager .up.|> Model
+ModelManager *-right-> "1" UserPrefs
```

Comment:
I'm not sure about the use of composition in this case. Also, it seems inconsistent with a class diagram earlier on in the DG where there is no composition between `ModelManager` and `UserPrefs`.  
An argument can certainly be made both ways, but I think it's better to leave out composition in cases where we are not sure.

Reply:
Good point, I will remove the composition here then!